web/src/compose_notifications.js

```diff
@@ -47,12 +48,33 @@ export function notify_above_composebox(
             link_text,
         }),
     );
-    // We pass in include_unmute_banner as false because we don't want to
-    // clear any unmute_banner associated with this same message.
-    compose_banner.clear_message_sent_banners(false);
+    // No need to clear banners that help to scroll to view or narrow to recipient
+    // while displaying the automatic new visibility policy banner.
+    if (!automatic_new_visibility_policy_banner) {
+        // We pass in include_unmute_banner as false because we don't want to
+        // clear any unmute_banner associated with this same message.
+        compose_banner.clear_message_sent_banners(false);
+    }
     compose_banner.append_compose_banner_to_banner_list($notification, $("#compose_banners"));
 }
 
+export function notify_automatic_new_visibility_policy(new_visibility_policy) {
+    let banner_text;
+    if (new_visibility_policy === user_topics.all_visibility_policies.FOLLOWED) {
+        banner_text = $t({defaultMessage: "You are now following this topic."});
+    } else {
+        banner_text = $t({defaultMessage: "You have now unmuted this topic."});
+    }
```

Comment:
This phrasing feels odd to me, since the user hasn't specifically unmuted the topic -- the app unmuted it for them.

The passive voice isn't ideal, but maybe: "stream > topic has been unmuted. [Change setting]()" ?

Reply:
We could try something briefer like `Now following #stream>topic. Change setting` or `Unmuted #stream>topic. Change setting`? Not sure.

Reply:
Sure, yeah, we can try those briefer versions and see if anyone complains that they aren't clear enough.

Reply:
[Updated banners](https://github.com/zulip/zulip/pull/27229#issuecomment-1768205166)